pom.xml

```diff
@@ -40,32 +47,32 @@
     <java.version>11</java.version>
     <!-- Testing -->
     <jacoco.version>0.8.5</jacoco.version>
-    <powermock.version>2.0.4</powermock.version>
-    <easymock.version>4.1</easymock.version>
+    <powermock.version>2.0.7</powermock.version>
+    <easymock.version>4.2</easymock.version>
     <!-- Build -->
     <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-    <maven-source-plugin.version>3.2.0</maven-source-plugin.version>
-    <maven-javadoc-plugin.version>3.1.1</maven-javadoc-plugin.version>
+    <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
+    <maven-javadoc-plugin.version>3.2.0</maven-javadoc-plugin.version>
     <maven-gpg-plugin.version>1.6</maven-gpg-plugin.version>
-    <maven-surefire-plugin.version>2.22.1</maven-surefire-plugin.version>
-    <maven-failsafe-plugin.version>2.22.1</maven-failsafe-plugin.version>
-    <maven-enforcer-plugin.version>3.0.0-M2</maven-enforcer-plugin.version>
-    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
+    <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
+    <maven-failsafe-plugin.version>3.0.0-M4</maven-failsafe-plugin.version>
+    <maven-enforcer-plugin.version>3.0.0-M3</maven-enforcer-plugin.version>
+    <maven-release-plugin.version>3.0.0-M1</maven-release-plugin.version>
     <maven-scm-provider-gitexe.version>1.11.2</maven-scm-provider-gitexe.version>
-    <exists-maven-plugin.version>0.0.6</exists-maven-plugin.version>
+    <exists-maven-plugin.version>0.3.0</exists-maven-plugin.version>
     <github-release-plugin.version>1.4.0</github-release-plugin.version>
     <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
     <!-- Spring -->
-    <spring.version>5.2.1.RELEASE</spring.version>
-    <spring-boot.version>2.2.1.RELEASE</spring-boot.version>
+    <spring.version>5.2.6.RELEASE</spring.version>
+    <spring-boot.version>2.3.0.RELEASE</spring-boot.version>
   </properties>
 
   <dependencies>
     <!-- Docker client -->
     <dependency>
       <groupId>com.github.docker-java</groupId>
       <artifactId>docker-java</artifactId>
-      <version>3.1.1</version>
+      <version>3.2.1</version>
```

Comment:
Gibt's eigentlich einen Grund dafür, dass manche Versionen in den Properties geführt werden und andere nicht? Sollte man das nicht mal vereinheitlichen?

Reply:
Passe ich mal an.